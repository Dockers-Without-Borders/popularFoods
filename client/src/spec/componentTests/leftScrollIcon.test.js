import React from 'react';
import { shallow } from 'enzyme';
import LeftScrollIcon from '../../components/carousel/icons/leftScrollIcon.jsx';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

describe('<LeftScrollIcon/>', () => {
    it('should render', () => {
        const component = shallow(<LeftScrollIcon />);
        expect(component.exists()).toEqual(true);
    });
    it('should render properly - snapshot version', () => {
        const component = renderer.create(<LeftScrollIcon />);
        expect(component).toMatchSnapshot();
    });
})