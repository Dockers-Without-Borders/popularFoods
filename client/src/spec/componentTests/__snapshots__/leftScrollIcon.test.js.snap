// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LeftScrollIcon/> should render properly - snapshot version 1`] = `
<svg
  className="prefix__icon_svg"
  height={24}
  width={24}
>
  <path
    d="M14.475 18.364l1.414-1.414L10.94 12l4.95-4.95-1.415-1.414L8.11 12l6.365 6.364z"
  />
</svg>
`;
